porxy/docker-entrypoint.sh

```diff
@@ -1,10 +1,3 @@
 #!/bin/bash
 
-ip=$(ip -4 addr show docker0 2>/dev/null | grep -Po 'inet \K[\d.]+')
-if [ -z "$ip" ];then
-  ip=$(ping -c 1 host.docker.internal | head -n 1 | sed -r 's/PING\shost\.docker\.internal\s\(([^\)]+).*/\1/')
-fi
-
-echo -e "${ip}\tmaster-account.3scale.localhost">>/etc/hosts
-
```

Comment:
`bin/porta` should handle `master-account.3scale.localhost`. Needs to be tested whether it works well on mac.